exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity >= moreStudentsThanMentors; // finish this statement
```

Comment:
you are comparing a number (roomMaxCapacity) to a boolean (moreStudentsThanMentors). how can you check whether all the students and mentors is less than the capacity?